mu-plugins/blocks/global-header-footer/src/header/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Disabled } from '@wordpress/components';
+import { registerBlockType } from '@wordpress/blocks';
+import ServerSideRender from '@wordpress/server-side-render';
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+
+const variations = [
```

Comment:
It would be nice if these came from the `block.json` attributes. Just noting that, I recognize how this is limited by the translations.

Reply:
There's also the `isActive` callback, so the variations need to be set in JS, not JSON. We could move the variation definitions to a separate shared folder later, but I didn't think the added overhead was worth it yet.